StripeCryptoOnramp/StripeCryptoOnramp/Source/Internal/API Bindings/KYCDataCollectionResponse.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  KYCDataCollectionResponse.swift
+//  StripeCryptoOnramp
+//
+//  Created by Michael Liberatore on 8/4/25.
+//
+
+import Foundation
+
+/// Codable model representing a response from the `/v1/crypto/internal/kyc_data_collection` endpoint.
+struct KYCDataCollectionResponse: Codable {
```

Comment:
`Codable`, not `Decodable`, so that it's easier to generate the response in tests.